keps/sig-release/3000-artifact-distribution/README.md

```diff
@@ -0,0 +1,135 @@
+
+This KEP should cover the policy and distribution mechanisms we will put in place to allow creating a globally distributed, multi-cloud and country solution.
+
+## Motivation
+
+Currently we push to a single provider, and distributing to the rest of community comes at great cost nearing $150k/month (mostly egress) in donations.
+
+Additionally, some of our community members are unable to access the official release artifacts due to country level firewalls that do not them connect to Google services.
+
+Ideally we can dramatically reduce cost and allow everyone in the world to download the artifacts released by our community.
+
+### Goals
+
+A policy and procedure for use by SIG Release to promote container images and release binaries to multiple registries and mirrors.
+
+A solution to allow redirection to appropriate mirrors to lower cost and allow access from any cloud or country globally.
+
+### Non-Goals
+
+Anything related to creation of artifacts, bom, digital signatures.
+
+## Proposal
+
+There are two intertwined concepts that are part of this proposal.
+
+First, the policy and procedures to promote/upload our artifacts to multiple providers. Our existing processes upload only to GCS buckets. Ideally we extend the existing software/promotion process to push directly to multiple providers. Alternatively we use a second process to synchronize artifacts from our existing production buckets to similar constructs at other providers.
+
+Additionally we require a registry and artifact url-redirection solution to the local cloud provider or country.
+
+### User Stories
+
+#### SIG Release - Image Promotion
+
+```feature
+As a SIG Release volunteer
+I want to promote our binaries/images to multiple clouds
+
+Given a promotion / manifest
+When my PR is merged
+Then the promotion process occurs
+```
+
+#### Cloud Customer - Installing K8s via kubeadm
+
+```feature
+As a CLOUD end-user
+I want to install Kubernetes
+
+Given some compute resources at CLOUD
+When I use kubeadm to deploy Kubernetes
+Then I will be redirected to a local CLOUD registry
+```
+
+### Notes/Constraints/Caveats
+
+The primary purpose of the KEP is getting consensus on the agreed policy and procedure to unblock our community and move forward together.
+
+There has been a lot of activity around the technology and tooling for both goals, but we need shared agreement on policy and procedure first.
+
+### Risks and Mitigations
+
+This is the primary pipeline for delivering Kubernetes worldwide. Ensuring the appropriate SLAs and support as well as artifact integrity is crucial.
+
+## Design Details
+
+### Artifact Promotion
+
+#### Policy
+
+(more details needed, #sig-release-eng?)
+
+#### Process
+
+Currently the promotion process is primarily driven by the CIP/[promo-tool#kpromo](https://github.com/kubernetes-sigs/promo-tools#kpromo)?
+
+### Artifact Distribution
+
+#### Policy
+
+#### Process
```

Comment:
We will need to detail how we turn up the new DNS name and redirector and how we plan to convert users of old GCR name into the new name.

Reply:
Members of sig-k8s-infra have made PRs against projects like kops and Kubernetes to change the defaults.
There's also e2e testing through changing the domain for various jobs running in Prow.